file/readfile_test.go

```diff
@@ -194,15 +199,24 @@ func Test_getContent(t *testing.T) {
 			},
 			wantErr: false,
 		},
+		{
+			name:    "file with bad environment variable",
+			args:    args{[]string{"testdata/bad-env-var/file.yaml"}},
+			wantErr: true,
+		},
```

Comment:
Test for the unset var suggestion earlier:

```suggestion
		},
    {
        name:    "unset environment variable in file",
        args:    args{[]string{"testdata/file.yaml"}},
        wantErr: true,
    },
```